conf/application.conf

```diff
@@ -176,7 +176,7 @@ external-url {
 accessibility-statement.service-path = "http://localhost:12346/accessibility-statement/get-a-customs-subscription-and-eori-number"
 
 services-config {
-    list = "atar, gagmr, cds, eori-only, ctc, gbss, route1, ars"
+    list = "atar, gagmr, cds, eori-only, ctc, gbss, route1, ars, ivd, ndrc"
```

Comment:
esc is missing from this list

Reply:
Adding a comment so that I can set this to request changes